README.md

```diff
@@ -3,230 +3,161 @@
 
 ## Prerequisites
 
-The following assumes that you have completed the steps for setting up your local environment as well as creating an account and project.  If you have not done this you must follow the instructions here:
+* [Your Lightbend Cloudstate Account](https://docs.lbcs.io/gettingstarted/account.html)
+* Install [csctl](https://docs.lbcs.io/getting-started/set-up-development-env.html)
 
-* [Setting Up your Machine](https://docs.lbcs.dev/gettingstarted/setup.html)
-   * as well as the [Developer prerequisites](https://docs.lbcs.dev/developing/developing.html#prerequisites)
-   * Install [nvm](https://github.com/nvm-sh/nvm#install--update-script) (node version manager)
-      * We recommend v0.34.0 or later.  (Check with `nvm --version`)
-   * Install [npm](https://www.npmjs.com/get-npm) (node package manager)
-      * We recommend v6.14.3 or later.  (Check with `npm -v`)
-   * You also need to install the protobuf compiler.
-      * We recommend using v3.0.0 or later.  (Check with `protoc --version`)
-      * Mac OS X `brew install protobuf`
-      * Linux `sudo apt install protobuf-compiler`
-      * Or [alternatively](https://developers.google.com/protocol-buffers/docs/downloads) (src and bins)
-* [Your Lightbend Cloudstate Account](https://docs.lbcs.dev/gettingstarted/account.html)
-* [Creating a Project](https://docs.lbcs.dev/gettingstarted/project.html)
+## Deploying
 
-## Sample application layout
+The following steps use `csctl` to deploy both the [shopping-cart frontend
+service](https://github.com/cloudstateio/samples-ui-shoppingcart) and the
+CloudState shopping-cart service defined in this repo. We'll be using the
+pre-built Docker images for the frontend and the shopping-cart, but feel free to
+build and use your own Docker images where appropriate.
 
-The sample application consists of 2 services:
-* A stateless service `frontend`
-* A stateful Entity based service `shopping-cart`
+### 1. Login
 
-Additionally:
-* A `cloudstate` directory that contains proto definitions needed.
-* A `deploy` directory that contains the deployment yaml files.
+```shell
+$ csctl auth login
+```
 
-### Quick Install
+### 2. Create a new project
 
-All the latest docker images are available publicly at `lightbend-docker-registry.bintray.io/cloudstate-samples`.
+```shell
+$ csctl projects new sample-shopping-cart "Shopping Cart Sample"
+```
 
-To deploy the shopping-cart application as is, connect to your kubernetes environment and do the following.
+### 3. List projects
 
-```bash
-$ cd deploy
-$ kubectl apply -f . -n <project-name>
-# verify stateful store
-$ kubectl get -n <project-name> statefulstore
```

Comment:
As far as I know, we would like to provide 2 sets of instructions: one using `csctl` for Lightbend Cloudstate hosting and one using `kubectl` for self-hosted open-source Cloudstate. So these instructions are still needed.

Reply:
OK, I can definitely add that back in.

Reply:
@rstento how would we like to organize those 2 sets of instructions? I like tabs like here https://docs.lbcs.dev/administration/log.html We can use them when we eventually convert the instructions to AsciiDoc.
@michaelpnash has mentioned that we may also want to use tables.

Reply:
One option would be to put the self-hosted instructions in the deploy directory and link to that from the top-level README.

Reply:
@vkorenev & @dudleycarr, I like the idea of putting the self-hosted instructions in a separate file that is linked to from the readme for now. Once we get the asciidoc set up, we can play around with the best presentation, perhaps tabs, perhaps a table. Sound OK?

Reply:
@dudleycarr @rstento, yes, I think it is good to place self-hosted instructions in a separate file. Actually, there are 3 logical sections which we could put in separate files:
- local development and building image(s)
- deploying on Lightbend Cloudsate
- deploying on generic k8s cluster